Update documentation (#107)
OpenSCAP/OVAL-visualization-as-graph: README.md

## Installation

### Enterprise Linux

> **Before install enable Extra Packages for Enterprise Linux (EPEL). How to enable EPEL is written [here](https://fedoraproject.org/wiki/EPEL).**

Reviewer: ```suggestion
**Before installing, enable Extra Packages for Enterprise Linux (EPEL). Learn how to enable EPEL in [EPEL documentation](https://fedoraproject.org/wiki/EPEL).**
```
